### DISCUSSION TITLE
Section headers for manuscript notebooks.

### DISCUSSION THREAD

**Discussion by jma1991** [quarto-dev/quarto-cli#9378] [Q&A]
Is it possible to add section headers to the notebooks area of a manuscript project? My use-case is that I often have 5-10 notebooks for each phase of an analysis and I'd like to group them accordingly (e.g., Data Processing, Data Visualization). Right now, I just have a big list of notebooks to the side of the main article and it's a bit hard to find certain notebooks.

**Comment by cscheid** [comment]
In the future, please file these in quarto-cli under discussions.
